src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-graphql_query.py

```diff
@@ -0,0 +1,19 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2005-2023, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License (version 2
+# or later) with exception for distributing the bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#
+# SPDX-License-Identifier: (GPL-2.0-or-later WITH Bootloader-exception)
+#-----------------------------------------------------------------------------
+"""
+PyInstaller hook file for graphql_query. Tested with version 1.0.3.
+"""
+
+from PyInstaller.utils.hooks import collect_data_files
+
+hiddenimports = ["graphql_query"]
```

Comment:
In order for the hook to be ran, the module already had to be discovered. So this hidden import is definitely not required.